src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/EnvelopeProcessor.java

```diff
@@ -109,6 +109,7 @@ public Envelope getEnvelopeByFileAndContainer(String container, String zipFileNa
         .orElse(null);
     }
 
+    // TODO: move to separate class.
```

Comment:
hmm ... I'm not sure what the policy with Todo's are.
In previous projects to avoid Tech dept going unnoticed we created a jira ticket involved in the todo ..
i.e. //TODO: RPE-XXX move to a separate class